modules/persistent-storage-csi-sc-multiple-none.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-sc-manage
+//
+:_content-type: PROCEDURE
+[id="persistent-storage-csi-sc-multiple-none_{context}"]
+= Absent or multiple default storage classes
+
+== Multiple default storage classes
+Multiple default storage classes can occur if you mark a non-default storage class as default and do not unset the existing default storage class, or you create a default storage class when a default storage class is already present. With multiple default storage classes present, any persistent volume claim (PVC) requesting the default storage class (`pvc.spec.storageClassName`=nil) gets the most recently created default storage class, and the administrator receives an alert that there are multiple default storage classes.
+
+== Absent default storage class
+There are two possible scenarios where PVCs can attempt to use a non-existent default storage class:
+
+* An administrator removes the default storage class or marks it as non-default, and then a user creates a PVC requesting the default storage class.
+
+* During installation, the installer creates a PVC for the image registry, requesting the default storage class, which has not yet been created. 
```

Comment:
it's not necessarily the image registry this is just an example. I would just go with 
the installer creates a PVC requesting the default storage class, which has not yet been created.